docs/guides/nrfconnect_examples_cli.md

```diff
@@ -0,0 +1,74 @@
+# Using CLI in nRF Connect SDK examples
```

Comment:
I don't see other guides than nrfconnect ones, so maybe instead of guides it would make sense to have docs/nrfconnect/ folder, and then structure it as it grows?